appinventor/docs/reference/other/sizes.html

```diff
@@ -434,177 +434,28 @@
-        <b>
-         If a HorizontalArrangement's Width property is set to Automatic, any
-                components whose Width properties are set to Fill Parent will behave as if they
-                were set to Automatic.
-        </b>
-       </p>
-       <p>
-        If a HorizontalArrangement's Width property is set to Fill Parent or specified in
-                pixels, any components whose Width properties are set to Fill Parent will equally
-                take up the width not occupied by other components.
-       </p>
-       <h3>
-        TableArrangement
-       </h3>
-       <p>
-        In a TableArrangement, components are arranged in a grid of rows and columns, with
-                not more than one component visible in each cell.
-        <b>
-         If multiple components occupy
-                the same cell, only the last one will be visible.
-        </b>
-       </p>
-       <p>
-        Within each row, components are vertically center-aligned.
-       </p>
-       <p>
-        The width of a column is determined by the widest component in that column. When
-                calculating column width, the automatic width is used for components whose Width
-                property is set to Fill Parent.
-        <b>
-         However, each component will always fill the
-                full width of the column that it occupies.
-        </b>
-       </p>
-       <p>
-        The height of a row is determined by the tallest component in that row whose Height
-                property is not set to Fill Parent. If a row contains only components whose Height
-                properties are set to Fill Parent, the height of the row is calculated using the
-                automatic heights of the components.
-       </p>
-       <h3>
-        Canvas
-       </h3>
-       <p>
-        Only Balls and ImageSprites can be placed in a Canvas. Each Ball and ImageSprite is
-                positioned on the canvas based on its X and Y properties. A Ball's width and height
-                is always the diameter of the ball. The actual width/height of an ImageSprite whose
-                Width/Height property is set to Automatic or Fill Parent will be the width/height
-                of the image.
-       </p>
-                    <!-- Content -->
+
+
+       <!-- Here's where we insert the link to the Google Doc that holds the page contents, so people
+            can comment and edit, together with an iframe that holds the ame doc.  Apparently the
+            iFrame itself is not editable, which is why we link to the Doc twice. -->
+
+       <a href="https://docs.google.com/document/d/1S1BBmFanf0KOD_W6HG35YEGwoAnlW7FZeq0cuMcBdZ0/edit#heading=h.3jgqgjwwqxts"
+	   target="_blank">
+           Click here for a version of this page on which you can comment.</a>
+
+       <iframe width="100%" height="800"
+	       src="https://docs.google.com/document/d/1S1BBmFanf0KOD_W6HG35YEGwoAnlW7FZeq0cuMcBdZ0/edit#heading=h.3jgqgjwwqxts">
```

Comment:
Again, use the published version in the iframe. Also note. I don't have access to this document, so I see a permission required screen inside the iframe.